<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HospitalGuideRobot</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1021</width>
      <height>711</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <item row="0" column="0" colspan="3">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_hospitalName">
         <property name="text">
          <string>XX Hospital</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_currentTime">
         <property name="text">
          <string>YYYY-MM-DD HH:MM:SS</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QListWidget" name="listWidget_navigation">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>135</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>Patient Info</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Navigation Guidelines</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Other</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QStackedWidget" name="stackedWidget_mainDisplay">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>32</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="page_patientInfo">
        <widget class="QWidget" name="gridLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>861</width>
           <height>661</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="3" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Registration:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0" colspan="8">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="5">
           <widget class="QComboBox" name="comboBox_gender">
            <item>
             <property name="text">
              <string notr="true">Male</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">Female</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">Other</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit_name">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_name__">
            <property name="text">
             <string>Full Name:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QLabel" name="label_5">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>This is your Info:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QLabel" name="label_gender__">
            <property name="text">
             <string>Gender:</string>
            </property>
           </widget>
          </item>
          <item row="6" column="6">
           <widget class="QPushButton" name="pushButton_nav">
            <property name="text">
             <string>Nav</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_dob__">
            <property name="text">
             <string>Date of Birth:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="7">
           <widget class="QPushButton" name="pushButton_clear">
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
          <item row="0" column="6">
           <widget class="QPushButton" name="pushButton_enter">
            <property name="text">
             <string>Enter</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QDateEdit" name="dateEdit"/>
          </item>
          <item row="2" column="1" colspan="3">
           <widget class="QLabel" name="label_id">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>ID:</string>
            </property>
           </widget>
          </item>
          <item row="6" column="7">
           <widget class="QPushButton" name="pushButton_help">
            <property name="text">
             <string>Face</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1" colspan="2">
           <widget class="QLabel" name="label_reg">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Apartment:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="4" colspan="2">
           <widget class="QLabel" name="label_apart">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="3" column="6">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Date:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="7">
           <widget class="QLabel" name="label_date">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="4">
           <widget class="QWidget" name="cameraWidget" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>300</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="page_navigation">
        <widget class="QWidget" name="gridLayoutWidget_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>9</y>
           <width>911</width>
           <height>661</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="2" column="1">
           <widget class="SvgViewer" name="graphicsView_map">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>500</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>1000</width>
              <height>2000</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>The hospital floor plan is shown below, please retrieve your desired destination in the map. You can also search by category in the sidebar</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QTextBrowser" name="textBrowser_log"/>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="page_other">
        <widget class="QTabWidget" name="tabWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>861</width>
           <height>641</height>
          </rect>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_ai">
          <attribute name="title">
           <string> AI Helper</string>
          </attribute>
          <widget class="QWidget" name="gridLayoutWidget_5">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>851</width>
             <height>601</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_5"/>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_translation">
          <attribute name="title">
           <string>Translation Service</string>
          </attribute>
          <widget class="QWidget" name="gridLayoutWidget_4">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>831</width>
             <height>561</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="2" column="1">
             <widget class="QComboBox" name="comboBox_from">
              <item>
               <property name="text">
                <string>English</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Chinese</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="0" column="0" colspan="6">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="2" column="3">
             <widget class="QComboBox" name="comboBox_to">
              <item>
               <property name="text">
                <string>Chinese</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>English</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QLabel" name="label_4">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>To:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="4">
             <widget class="QPushButton" name="pushButton_4">
              <property name="text">
               <string>Enter</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_3">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>From:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="5">
             <widget class="QPushButton" name="pushButton_3">
              <property name="text">
               <string>Clear</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="6">
             <widget class="QPlainTextEdit" name="plainTextEdit_translate"/>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionFull_Screen"/>
    <addaction name="actionMinimize_Screen"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionFull_Screen">
   <property name="text">
    <string>Full Screen</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionMinimize_Screen">
   <property name="text">
    <string>Minimize Screen</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SvgViewer</class>
   <extends>QGraphicsView</extends>
   <header>svgviewer.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>listWidget_navigation</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>stackedWidget_mainDisplay</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>348</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>plainTextEdit_translate</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
